src/grdfilter.c

```diff
@@ -584,72 +584,79 @@ GMT_LOCAL struct GMT_GRID *init_area_weights (struct GMT_CTRL *GMT, struct GMT_G
+		"Append +h to select high-pass filtering [Default is low-pass filtering]. "
+		"Filters are isotropic.  For rectangular filtering append /<width2> (requires -Dp|0). "
+		"Convolution filters:");
+	GMT_Usage (API, 3, "b: Boxcar : a simple averaging of all points inside filter domain.");
+	GMT_Usage (API, 3, "c: Cosine arch : a weighted averaging with cosine arc weights.");
+	GMT_Usage (API, 3, "g: Gaussian : weighted averaging with Gaussian weights.");
+	GMT_Usage (API, 3, "f: Custom : give grid file with custom filter weights (requires -D0).");
+	GMT_Usage (API, 3, "o: Operator : Give grid file with custom operator weights (requires -D0).");
+	GMT_Usage (API, -2, "Geospatial filters:");
+	GMT_Usage (API, 3, "h: Histogram-based mode estimator. Append <width>/<bin> and optional modifiers. "
+		"Data inside filter domain is binned using the <bin> increment:");
+	GMT_Usage (API, 4, "+c Center the bins on multiples of <bin> [Default has bin edges as multiples].");
+	GMT_Usage (API, 4, "+l Return the lowest mode if multiple modes are found [return average].");
+	GMT_Usage (API, 4, "+u Return the uppermost mode if multiple modes are found [return average].");
+	GMT_Usage (API, 3, "l: Lower : return minimum of all points.");
+	GMT_Usage (API, 3, "L: Lower+ : return minimum of all positive points.");
+	GMT_Usage (API, 3, "m: Median : return the median (50%% quantile) value of all points. "
+		"Append +q<quantile> to select another quantile (in 0-1 range) [0.5].");
+	GMT_Usage (API, 3, "p: Maximum likelihood probability estimator - return mode of all points. "
+		"We approximate the mode using the Least Median of Squares estimator.");
 #if 0
-	GMT_Message (API, GMT_TIME_NONE, "\t        Append +c<span> to select a different LMS span, in percent [50].\n");
+	GMT_Usage (API, 4, "+c Select a different LMS <span>, in percent [50].");
 #endif
-	GMT_Message (API, GMT_TIME_NONE, "\t        Append +l to return the lowest mode if multiple modes are found [return average].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t        Append +u to return the uppermost mode if multiple modes are found [return average].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     u: Upper : return maximum of all points.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     U: Upper- : return maximum of all negative points.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     If <width> is a grid it must match output domain and registration\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-G Set output filename for filtered grid.\n");
+	GMT_Usage (API, 4, "+l Return the lowest mode if multiple modes are found [return average].");
+	GMT_Usage (API, 4, "+u Return the uppermost mode if multiple modes are found [return average].");
+	GMT_Usage (API, 3, "u: Upper : return maximum of all points.");
+	GMT_Usage (API, 3, "U: Upper- : return maximum of all negative points.");
+	GMT_Usage (API, -2, "If <width> is a grid it must match the output domain and registration.");
```

Comment:
Which width is meant by `<width>`? `<filter_width>[/<width2>]`

Reply:
Revised to use <width> and <width2> instead of the unnecessary filter_ prefix.